tools/dependencies/checkLicense.sh

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 
-./mvnw dependency:copy-dependencies -DincludeScope=runtime -DoutputDirectory=/tmp/seatunnel-dependencies
+./mvnw -pl '!seatunnel-examples,!seatunnel-examples/seatunnel-spark-examples,!seatunnel-examples/seatunnel-flink-examples' dependency:copy-dependencies -DincludeScope=runtime -DoutputDirectory=/tmp/seatunnel-dependencies
```

Comment:
can you try to add a plugin declaration in `seatunnel-examples` with `skip=true`, so when people add new examples they don't need to modify this file.

in file `seatunnel/seatunnel-examples/pom.xml`, the plugin should be extended by all children modules

```xml
<build>
  <plugins>
    <plugin>
      <groupId>org.apache.maven.plugins</groupId>
      <artifactId>maven-dependency-plugin</artifactId>
      <configuration>
        <skip>true</skip>
      </configuration>
    </plugin>
....
```

Reply:
Good idea.

Reply:
Good idea.thx:)